vpdq/cpp/regtest.py

```diff
@@ -1,9 +1,17 @@
 # Copyright (c) Meta Platforms, Inc. and affiliates.
 
 import subprocess
-import os
 import sys
 import argparse
+from pathlib import Path
+import glob
+
+DIR = Path(__file__).parent
+VPDQ_DIR = DIR.parent
+SAMPLE_VIDEOS_DIR = VPDQ_DIR.parent / "tmk/sample-videos"
+SAMPLE_HASHES_DIR = VPDQ_DIR / "sample-hashes"
+OUTPUT_HASHES_DIR = VPDQ_DIR / "output-hashes"
+EXEC_DIR = VPDQ_DIR / "cpp/build"
```

Comment:
(nit): Since we use as these as the defaults for argparse, and not definitively, it might be helpful to prefix these with `DEFAULT_` so a casual future author doesn't reference them directly.

(ignoreable, alt): One reason to put these in get_argparse rather than as globals here is it makes it harder to make that mistake. I think it's harmless, with the above fix, but you could hide these in `get_argparse` or a helper function to try and avoid it.

Reply:
Sure, I moved the parameters that be changed into get_argparse(). I only add DEFAULT to those because the others can't be changed. If parameters are added to change any of those as well they can be moved into get_argparse() and have a DEFAULT prefix.